de-lib/src/main/java/org/iplantc/de/apps/widgets/client/presenter/AppLaunchPresenterImpl.java

```diff
@@ -82,6 +82,21 @@ public void onSuccess(AppTemplate result) {
         }
     }
 
+    private class QuickLaunchTemplateCallback extends AppTemplateCallback {
+        private AppWizardConfig config;
+
+        public QuickLaunchTemplateCallback(AppWizardConfig config) {
+            this.config = config;
+        }
+
+        @Override
+        public void onFailure(Integer statusCode, Throwable caught) {
+            announcer.schedule(new ErrorAnnouncementConfig(appearance.quickLaunchDeleted(), true, 0));
```

Comment:
👍 I think it's a good idea to leave this error message displayed until dismissed.

Especially if someone is clicking a QL link from some other site, and they are still getting their bearings in the DE after the auto-dismiss timeout occurs. 😵